linux/kernel/building.md

```diff
@@ -125,6 +125,11 @@ If you are using a 32-bit operating system (for example, our Raspberry Pi Deskto
 
 `sudo apt install zlib1g-dev:amd64`
 
+If you are using CCache and a CI environment, instruct it to not use compiler's mtime for cache ID calculations.
```

Comment:
"and a CI environment" -> "(e.g. in a CI environment)" ?
"use compiler's" -> "use the compiler's" ?

Reply:
The CI environment is required, too, so no "e.g.". The problem only appears if you clone the tools anew each build.
Fine by me otherwise.